vm/src/hint_processor/builtin_hint_processor/bootloader_hints.rs

```diff
@@ -261,6 +261,21 @@ pub fn import_packed_output_schemas() -> Result<(), HintError> {
     Ok(())
 }
 
+/// Implements
+/// %{ assert isinstance(packed_output, CompositePackedOutput) %}
+pub fn assert_is_composite_packed_output(
+    exec_scopes: &mut ExecutionScopes,
+) -> Result<(), HintError> {
+    let packed_output: PackedOutput = exec_scopes.get(vars::PACKED_OUTPUT)?;
+
+    match packed_output {
+        PackedOutput::Composite(_) => Ok(()),
+        other => Err(HintError::CustomHint(
+            format!("Expected composite packed output, got {:?}", other).into_boxed_str(),
```

Comment:
Does the error message matter here?

Reply:
No, it just makes it slightly easier to debug if something goes wrong.